spec/presenters/period_contribution_calculator_presenter_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe Wpcc::PeriodContributionCalculatorPresenter do
+  subject do
+    described_class.new(pcc, view_context: view_context)
+  end
+
+  let(:pcc) { Wpcc::PeriodContribution.new }
+  let(:view_context) { double(:view_context) }
+
+  before do
+    allow(subject).to receive(:view_context)
+      .and_return(ActionController::Base.new.view_context)
+  end
+
+  describe '#title' do
+    it 'converts the name to a more readable format' do
+      pcc.name = 'april_2017_march_2018'
+      expect(subject.title).to eq('Now')
+    end
+  end
+
+  describe '#employee_percent' do
+    let(:pcc) { double(employee_percent: 1.0) }
```

Comment:
I feel like this is a different object to `let(:pcc) { Wpcc::PeriodContribution.new }`..? again feels to me like the name of the variable `pcc` could be a bit clearer